src/types/sheetElementDescriptors/label.ts

```diff
@@ -5,5 +5,5 @@ import { GenericSheetElementDescriptor } from "./generic";
  */
 export interface LabelElementDescriptor extends GenericSheetElementDescriptor {
   for: string;
-  text: string;
+  text: string | string[];
```

Comment:
Make string | string[] it's own variable type within GenericSheetElementDescriptor and use that for all values that can be variables instead